homeassistant/components/ubus/device_tracker.py

```diff
@@ -110,7 +110,7 @@ def get_device_name(self, device):
 
     async def async_get_extra_attributes(self, device: str) -> dict[str, str]:
         """Return the host to distinguish between multiple routers."""
-        return {"host": self.host}
+        return {"host": self.host, "signal": self.last_results[device]["signal"]}
```

Comment:
What kind of value is signal? What does it represent?

Reply:
It is the signal strength of the WiFi signal received by the WiFi interface. 
Could even be rssi in dBm.

Reply:
That should be implemented as a separate sensor entity.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes